fast64_internal/f3d/f3d_material.py

```diff
@@ -409,18 +409,20 @@ def ui_geo_mode(settings, dataHolder, layout, useDropdown):
             icon="TRIA_DOWN" if dataHolder.menu_geo else "TRIA_RIGHT",
         )
     if not useDropdown or dataHolder.menu_geo:
+        disable_dependent = False  # Don't disable dependent props in world defaults
 
-        def indentGroup(parent: UILayout, textOrProp: Union[str, "F3DMaterialProperty"], isText: bool) -> UILayout:
+        def indentGroup(
+            parent: UILayout, textOrProp: Union[str, "F3DMaterialProperty"], isText: bool, enable=True
+        ) -> UILayout:
             c = parent.column(align=True)
             if isText:
                 c.label(text=textOrProp)
             else:
                 c.prop(settings, textOrProp)
-                if not getattr(settings, textOrProp):
-                    return None
```

Comment:
Enable should not be a parameter, enable should be coming from this, that's the whole point of this function, to enable/disable the group based on whether the head of the group is enabled.